minio/minio
# allow direct access to bucket from browser with limited permissions

## Description
Start minio server. Create a user with password. Start minio gateway with the previously give username & password  as accesskey & secretkey. An attempt to login to gateway in the browser will give an error like shown below:
API: WebListBuckets()
Time: 12:16:02 PDT 05/11/2020
RemoteHost: 192.168.1.2
Host: 192.168.1.2:9002
UserAgent: .....
Error: Prefix access is denied: /
       3: github.com/minio/cmd/web-handlers.go:2221:cmd.toWebAPIError()
       2: github.com/minio/cmd/web-handlers.go:2092:cmd.toJSONError()
       1: github.com/minio/cmd/web-handlers.go:333:cmd.(*webAPIHandlers).ListBuckets()



## Motivation and Context


## How to test this PR?
Create user. For a given bucket, access policy to user may have (minimum list) - ListBucket, GetBucket, PutObject, GetObject. Try to create a bucket. It should show the error.

## Types of changes

## Checklist:
